cities/schemas.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str | None = None
```

Comment:
The type annotation `str | None` is correct for Python 3.10 and later. If you are using an earlier version, consider using `Optional[str]` instead.